cli/tests/integrations/test_marathon.py

```diff
@@ -384,10 +384,35 @@ def test_update_invalid_request():
     assert returncode == 1
     assert stdout == b''
     stderr = stderr.decode()
-    assert stderr.startswith('Error while fetching')
+    assert stderr.startswith('Error on request')
     assert stderr.endswith('HTTP 400: Bad Request\n')
 
 
+def test_app_add_invalid_request():
+    path = os.path.join(
+        'tests', 'data', 'marathon', 'apps', 'app_add_400.json')
+    stderr = b"""Error on request [POST http://localhost:5080/marathon/v2/\
```

Comment:
Will this fail if the user runs the integration tests against a non localhost Marathon?